kubeflow/core/tf-job-operator.libsonnet

```diff
@@ -55,6 +55,45 @@
           singular: "tfjob",
           plural: "tfjobs",
         },
+        validation: {
+          openAPIV3Schema: {
+            properties: {
+              spec: {
+                properties: {
+                  tfReplicaSpecs: {
+                    properties: {
+                      Worker: {
+                        properties: {
+                          replicas: {
+                            type: "integer",
+                            minimum: 1,
```

Comment:
Please add comment explaining why we set minimum to 1.
Also please add a comment explaining why the validation doesn't extend to the pod template spec.

Reply:
OK, I will update it.